### PR TITLE
feat(daffio): daffio footer update

### PR DESCRIPTION
**Pull request by xelaint**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
`<daffio-footer>` style is better for site with more links.

## What is the new behavior?
Simple footer for minimal links.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

File: apps/daffio/src/app/core/footer/footer.module.ts [hunk 1]
```diff
@@ -30,10 +35,14 @@ import {
...
   ],
   declarations: [
-    DaffioFooterComponent
+    DaffioFullFooterComponent,
+    DaffioSimpleFooterComponent,
+    DaffioSubFooterComponent
   ],
   exports: [
-    DaffioFooterComponent
+    DaffioFullFooterComponent,
```
Comment (damienwebdev) [inline apps/daffio/src/app/core/footer/footer.module.ts]: This component is never used, remove it from both `declarations` and `exports`.

Comment (xelaint) [reply apps/daffio/src/app/core/footer/footer.module.ts]: Resolved in 8f671cf

**Comment by damienwebdev** [comment]
@xelaint there's a minor merge conflict with white space.

**Comment by xelaint** [comment]
@damienwebdev fixed conflict, need to be re-approved.

**Comment by xelaint** [comment]
@damienwebdev squished.

**Pull request merged**